compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -239,10 +205,6 @@ impl MetaItem {
         self.path == name
     }
 
-    pub fn is_value_str(&self) -> bool {
-        self.value_str().is_some()
```

Comment:
This was technically used by clippy, but only twice, and it seems weird to keep it in-tree when it's so trivial to write.